src/prefect/orion/api/flow_runs.py

```diff
@@ -175,6 +175,68 @@ async def read_flow_run_graph(
         )
 
 
+@router.post("/{id}/resume")
+async def resume_flow_run(
+    flow_run_id: UUID = Path(..., description="The flow run id", alias="id"),
+    db: OrionDBInterface = Depends(provide_database_interface),
+    response: Response = None,
+    flow_policy: BaseOrchestrationPolicy = Depends(
+        orchestration_dependencies.provide_flow_policy
+    ),
+    orchestration_parameters: dict = Depends(
+        orchestration_dependencies.provide_flow_orchestration_parameters
+    ),
+    api_version=Depends(dependencies.provide_request_api_version),
+) -> OrchestrationResult:
+    """
+    Resume a paused flow run.
+    """
+    async with db.session_context(begin_transaction=True) as session:
+        flow_run = await models.flow_runs.read_flow_run(session, flow_run_id)
+        state = flow_run.state
+
+        if state is None or state.type != "PAUSED":
+            result = OrchestrationResult(
+                state=None,
+                status=schemas.responses.SetStateStatus.ABORT,
+                details=schemas.responses.StateAbortDetails(
+                    reason="Cannot resume a flow run that is not paused."
+                ),
```

Comment:
Here I think we'd want to return a `REJECT` with the current state of the run?

Reply:
I think this really is an `ABORT`, the process asking for this transition is either some utility, a UI interface or CLI command. Those things can't do anything with the current state and can only pause or error out